src/python/grapl-template-generator/grapl_template_generator/grapl-templates/rust-grpc-service/{{cookiecutter.project_slug}}/proto/{{cookiecutter.snake_project_name}}.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package {{cookiecutter.service_name}};
+
+message GraplRequestMeta {
+    string clientId = 1;
+}
```

Comment:
I agree. Also, I'd imagine this would be tenant_uuid? I'm not entirely sure what this represents.
@colin-grapl  should we have some universal shared GraplRequestMeta or something?

Reply:
Yes, we should remove this.